binders/kafka-binder/spring-cloud-stream-binder-kafka-reactive/src/test/java/org/springframework/cloud/stream/binder/reactorkafka/ReactorKafkaBinderIntegrationTests.java

```diff
@@ -42,76 +44,86 @@
 import org.springframework.kafka.support.converter.MessagingMessageConverter;
 import org.springframework.kafka.support.converter.RecordMessageConverter;
 import org.springframework.kafka.test.EmbeddedKafkaBroker;
-import org.springframework.kafka.test.condition.EmbeddedKafkaCondition;
 import org.springframework.kafka.test.context.EmbeddedKafka;
 import org.springframework.kafka.test.utils.KafkaTestUtils;
 import org.springframework.messaging.Message;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
+ * Integration tests for {@link ReactorKafkaBinder}.
+ *
  * @author Soby Chacko
  * @author Gary Russell
+ * @author Chris Bono
  */
+@ExtendWith(SpringExtension.class)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
 @EmbeddedKafka(topics = { "uppercased-words", "lowercased-words" })
-public class ReactorKafkaBinderIntegrationTests {
-
-	private static final EmbeddedKafkaBroker embeddedKafka = EmbeddedKafkaCondition.getBroker();
+class ReactorKafkaBinderIntegrationTests {
```

Comment:
Throughout the PR I have moved to package-protected access modifier for tests and test methods as that is one of the JUnit5 features. On the surface it seems trivial but it promotes: 
1. the "Less is more" mindset
2. Principle of Least Privileged as a default mode of operation

Reply:
To be handled in #2371